server/controllers/startPage/startPage.test.ts

```diff
@@ -1,14 +1,50 @@
-import { mockReq, mockRes } from '../../middleware/testutils/mockRequestUtils'
+import type { Response } from 'express'
+import jwt from 'jsonwebtoken'
+import { mockReq } from '../../middleware/testutils/mockRequestUtils'
 import { startPage } from './startPage'
 
+function createToken(authorities: string[]) {
+  const payload = {
+    user_name: 'USER1',
+    scope: ['read', 'write'],
+    auth_source: 'nomis',
+    authorities,
+    jti: 'a610a10-cca6-41db-985f-e87efb303aaf',
+    client_id: 'clientid',
+  }
+
+  return jwt.sign(payload, 'secret', { expiresIn: '1h' })
+}
+function createResWithToken(
+  { authorities }: { authorities: string[] },
+  { currentPageId }: { currentPageId: string }
+): Response {
+  return {
+    token: createToken(authorities),
+    locals: {
+      flags: {
+        flagProbationAdmin: 'true',
```

Comment:
This work should be separate to the probationAdmin flag.  We can perhaps just delete this, but it might mean it's been linked where it shouldn't be.

Reply:
It isn't linked to probationAdmin, I added the flag there accidentally as a cut/paste job :O

Reply:
Done :D